roles/openstack-distro/tasks/osp.yml

```diff
@@ -16,3 +16,13 @@
   retries: 5
   with_items: "{{ dependent_packages }}"
   when: dependent_packages is defined
+
+- name: python post dependencies for project
+  pip:
+    name: "{{ item.name }}"
+    version: "{{ item.version|default(omit) }}"
+    extra_args: "{{ item.pip_extra_args|default(omit) }}"
+  with_items: "{{ openstack_source.python_post_deps }}"
```

Comment:
Need to also port global pip_extra_args logic from openstack-source to openstack-distro role.